ImageEditor/build.gradle

```diff
@@ -27,30 +29,40 @@ android {
 
     sourceSets {
         main.java.srcDirs += 'src/main/kotlin'
-        release.java.srcDirs += 'src/release/kotlin'
-        debug.java.srcDirs += 'src/debug/kotlin'
         test.java.srcDirs += 'src/test/kotlin'
         androidTest.java.srcDirs += 'src/androidTest/kotlin'
     }
 }
 
 dependencies {
-    def nav_version = "2.0.0"
+    def navComponentVersion = "2.0.0"
```

Comment:
We might want to consider using the `ext { .. }` approach which here (we use in the other modules). Wdyt?

Reply:
I considered using `def` here as navigation component was used just in this module. I find a good explanation [here](https://stackoverflow.com/a/56969539/193545).
> it lets you click on the variable and points exactly where it is defined in the file 

But for consistency, I'll convert it to `ext { .. }` approach.

Reply:
Changed here 386b270

Reply:
👍 Thanks, I think it's a good call. If we keep consistency, finding the declaration shouldn't be an issue.